src/Catalyst.Core.Modules.Kvm/CatalystSpecProvider.cs

```diff
@@ -0,0 +1,36 @@
+#region LICENSE
+
+/**
+* Copyright (c) 2019 Catalyst Network
+*
+* This file is part of Catalyst.Node <https://github.com/catalyst-network/Catalyst.Node>
+*
+* Catalyst.Node is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 2 of the License, or
+* (at your option) any later version.
+*
+* Catalyst.Node is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with Catalyst.Node. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#endregion
+
+using Catalyst.Abstractions.Types;
+using Nethermind.Core.Specs;
+
+namespace Catalyst.Core.Modules.Kvm
+{
+    public sealed class CatalystSpecProvider : ISpecProvider
+    {
+        public IReleaseSpec GenesisSpec => CatalystGenesisSpec.Instance;
+        public IReleaseSpec GetSpec(long blockNumber) => GenesisSpec;
+        public long? DaoBlockNumber => null;
+        public int ChainId => NetworkTypes.Dev.Id; // @TODO should we not be using protocol.common.network?
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Complete the task associated to this 'TODO' comment.](https://app.codacy.com/app/catalyst-network/Catalyst.Node/pullRequest?prid=4318201)